docs/diagram-tech-arq.plantuml

```diff
@@ -12,22 +12,26 @@ left to right direction
 
 APIGateway(apiGateway, "API Gateway", "External API Patterns")
 ElasticKubernetesService(creditCrytoService, "Credit Crypto Service", "Saga Pattern")
-Blockchain(blockchainService, "Core Service", "Smart Contract")
+ElasticKubernetesService(blockchainService, "Core", "Block chain client")
 ManagedStreamingforApacheKafka(kafka, "Kafka", "Message Style Pattern")
 Database(databaseBlockchain, "Database", "Postgres Database")
-ElasticKubernetesService(erpService, "ERP", "Last mile service")
+Blockchain(blockchain, "blockchain", "blockchain")
```

Comment:
this is the blockchain that blockchainservice will call in order to add a tx or add a contract. The blockchain can be any: ethereum, btc, ada and so on.